orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -64,25 +64,25 @@ public class OrchestrationClient {
   /** Default constructor. */
   public OrchestrationClient() {
     destinationSupplier =
-        () ->
-            new AiCoreService()
-                .getDestinationForDeploymentByScenario(
-                    AiCoreService.DEFAULT_RESOURCE_GROUP, DEFAULT_SCENARIO);
+        () -> new AiCoreService().getInferenceDestination().forScenario(DEFAULT_SCENARIO);
   }
 
   /**
    * Constructor with a custom destination, allowing for a custom resource group or otherwise custom
-   * destination.
+   * destination. The destination needs to be configured with a URL pointing to an orchestration
+   * service deployment. Typically, such a destination should be obtained using {@link
+   * AiCoreService#getInferenceDestination(String)}.
    *
    * <p>Example:
    *
    * <pre>{@code
-   * new OrchestrationClient(new AiCoreService().getDestinationForDeploymentByScenario("custom-rg", "orchestration"));
+   * new OrchestrationClient(new AiCoreService().getInferenceDestination("custom-rg").forScenario("orchestration"));
    * }</pre>
    *
    * @param destination The specific {@link HttpDestination} to use.
-   * @see AiCoreService#getDestinationForDeploymentByScenario(String, String)
+   * @see AiCoreService#getInferenceDestination(String)
    */
+  @Beta
```

Comment:
Not super confident about this, considering there are things like a "global" or "default" orchestration service planned, thus the Beta